app/models/katello/content_view_version.rb

```diff
@@ -428,6 +428,24 @@ def after_promote_hooks
       end
     end
 
+    def add_applied_filters!
+      applied_filters_and_rules = content_view.filters.map do |f|
+        {
+          filter: f,
+          rules: f.rules,
+          affected_repos: f.applicable_repos.map do |repo|
+            repo.slice(:id, :name, :label, :arch, :major, :minor,
+                       :content_type, :os_versions, :url, :content_id).merge(redhat: repo.redhat?, root: repo.root.id)
```

Comment:
```suggestion
                       :content_type, :os_versions, :url, :content_id).merge(redhat: repo.redhat?, root: repo.root.id, product: repo.product.slice(:id, :name, :label))
```

Reply:
Updated to add product id and label. Here's what it looks like now:

```
{"applied_filters"=>
  [{"rules"=>
     [{"id"=>3, "name"=>"bear", "version"=>nil, "created_at"=>"2023-03-31T14:37:26.548-04:00", "updated_at"=>"2023-03-31T14:37:26.548-04:00", "max_version"=>nil, "min_version"=>nil, "architecture"=>"", "content_view_filter_id"=>9}],
    "filter"=>
     {"id"=>9,
      "name"=>"prod",
      "repos"=>["needed_errata", "modular"],
      "content"=>"rpm",
      "inclusion"=>true,
      "created_at"=>"2023-03-31T14:36:24.750-04:00",
      "updated_at"=>"2023-03-31T14:37:08.196-04:00",
      "description"=>"",
      "organization"=>"Default_Organization",
      "content_view_id"=>7,
      "original_packages"=>true,
      "content_view_label"=>"cv11",
      "original_module_streams"=>false},
    "affected_repos"=>
     [{"id"=>1,
       "url"=>"https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/",
       "arch"=>"noarch",
       "name"=>"needed_errata",
       "root"=>1,
       "label"=>"needed_errata",
       "major"=>nil,
       "minor"=>nil,
       "redhat"=>false,
       "product"=>{"id"=>1, "label"=>"prod"},
       "content_id"=>"1679500443174",
       "os_versions"=>[],
       "content_type"=>"yum"},
      {"id"=>5,
       "url"=>"https://partha.fedorapeople.org/test-repos/pteradactyl/",
       "arch"=>"noarch",
       "name"=>"modular",
       "root"=>2,
       "label"=>"modular",
       "major"=>nil,
       "minor"=>nil,
       "redhat"=>false,
       "product"=>{"id"=>1, "label"=>"prod"},
       "content_id"=>"1679504607274",
       "os_versions"=>[],
       "content_type"=>"yum"}]},
   {"rules"=>
     [{"id"=>56,
       "types"=>[],
       "end_date"=>nil,
       "date_type"=>"updated",
       "errata_id"=>"RHEA-2013:0075",
       "created_at"=>"2023-04-05T09:49:07.999-04:00",
       "start_date"=>nil,
       "updated_at"=>"2023-04-05T09:49:07.999-04:00",
       "content_view_filter_id"=>10},
      {"id"=>57,
       "types"=>[],
       "end_date"=>nil,
       "date_type"=>"updated",
       "errata_id"=>"RHEA-2013:0070",
       "created_at"=>"2023-04-05T09:49:08.005-04:00",
       "start_date"=>nil,
       "updated_at"=>"2023-04-05T09:49:08.005-04:00",
       "content_view_filter_id"=>10}],
    "filter"=>
     {"id"=>10,
      "name"=>"needed_errata",
      "repos"=>["zoo", "modular"],
      "content"=>"erratum",
      "inclusion"=>false,
      "created_at"=>"2023-04-05T09:49:02.184-04:00",
      "updated_at"=>"2023-04-05T09:49:02.184-04:00",
      "description"=>"",
      "organization"=>"Default_Organization",
      "content_view_id"=>7,
      "original_packages"=>false,
      "content_view_label"=>"cv11",
      "original_module_streams"=>false},
    "affected_repos"=>
     [{"id"=>6,
       "url"=>"https://partha.fedorapeople.org/test-repos/zoo1/",
       "arch"=>"noarch",
       "name"=>"zoo",
       "root"=>3,
       "label"=>"zoo",
       "major"=>nil,
       "minor"=>nil,
       "redhat"=>false,
       "product"=>{"id"=>1, "label"=>"prod"},
       "content_id"=>"1679504669458",
       "os_versions"=>[],
       "content_type"=>"yum"},
      {"id"=>5,
       "url"=>"https://partha.fedorapeople.org/test-repos/pteradactyl/",
       "arch"=>"noarch",
       "name"=>"modular",
       "root"=>2,
       "label"=>"modular",
       "major"=>nil,
       "minor"=>nil,
       "redhat"=>false,
       "product"=>{"id"=>1, "label"=>"prod"},
       "content_id"=>"1679504607274",
       "os_versions"=>[],
       "content_type"=>"yum"}]}],
 "dependency_solving"=>false}
```